apps/volk_option_helpers.cc

```diff
@@ -188,6 +188,7 @@ void option_list::parse(int argc, char** argv)
                     } catch (std::exception& exc) {
                         throw std::exception();
                     };
+                    break;
```

Comment:
The fall-through here was a bug; it caused a blank line to be printed if the `-R, `-j`, or `-p` options were used with `volk_profile`.